website/release-notes/sprint-review-2414.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Wazo Platform 24.14 Released
+date: 2024-10-15T08:00:00
+author: wazoplatform
+category: Wazo Platform
+tags: [wazo-platform, development]
+slug: release-review-2414
+status: published
+---
+
+Hello Wazo Platform community!
+
+Here is a short review of the Wazo Platform 24.14 release.
+
+## Technical
+
+- **Call Logs**: Call logs now better handles call forwarding. The requested fields will contain the call's intended target information while destination fields
+  will contain the forwarded callee information
+
+## Ongoing Features
+
+- **Phone numbers**: We are working on a new phone number API that will be used in conjunction with incoming calls and the selection of outgoing caller ID
+
+For more details about the aforementioned topics, please see the [changelog](https://wazo-dev.atlassian.net/issues/?jql=project%3DWAZO%20AND%20fixVersion%3D24.14).
+
+See you at the next sprint review!
+
+## Resources
+
+- [Install Wazo Platform](/use-cases)
+- [Upgrade Wazo and Wazo Platform](/uc-doc/upgrade/). Be sure to read the
+  [breaking changes](/uc-doc/upgrade/upgrade_notes#24-14)
+
+<!-- truncate -->
```

Comment:
What is this line doing here?

Reply:
it's for the new website, for preview

Reply:
Hmm, not sure, it is part of the new template
Maybe @chartrandf has more info?

Reply:
Seems to be a docusaurus feature
https://docusaurus.io/docs/blog#blog-list

Reply:
Yes exactly, is to make a «clean excerpt» in the listing.